packages/client/e2e/test-a.spec.ts

```diff
@@ -281,7 +280,7 @@ test.afterEach(async ({ browser }) => {
     await closeAllContexts(browser);
 });
 
-test('Test A', async ({ browser }) => {
+test.only('Test A', async ({ browser }) => {
```

Comment:
We should not keep a test with `.only`.

I will suggest a PR that configures ESLint to disallow committing tests with a `.only`.

Reply:
Great idea